docs/style.css

```diff
@@ -0,0 +1,78 @@
+h3 {
+  text-align: center;
+}
```

Comment:
Instead of directly applying the rule to `h3`, make a utility class for this and add that class to the element.